migration/identity-migration/src/main/java/io/camunda/migration/identity/config/oidc/StaticEntities.java

```diff
@@ -0,0 +1,323 @@
+                    Set.of(PermissionType.READ)),
+                new CreateAuthorizationRequest(
+                    ownerId,
+                    ownerType,
+                    "*",
+                    AuthorizationResourceType.ROLE,
+                    AuthorizationResourceType.ROLE.getSupportedPermissionTypes())),
+            properties.getOperate() + ":read:*",
+            List.of(
+                new CreateAuthorizationRequest(
+                    ownerId,
+                    ownerType,
+                    "*",
+                    AuthorizationResourceType.MESSAGE,
+                    Set.of(PermissionType.READ)),
+                new CreateAuthorizationRequest(
+                    ownerId,
+                    ownerType,
+                    "*",
+                    AuthorizationResourceType.BATCH_OPERATION,
+                    Set.of(PermissionType.READ)),
+                new CreateAuthorizationRequest(
+                    ownerId,
+                    ownerType,
+                    OPERATE_RESOURCE_ID,
+                    AuthorizationResourceType.APPLICATION,
+                    Set.of(PermissionType.ACCESS)),
+                new CreateAuthorizationRequest(
+                    ownerId,
+                    ownerType,
+                    "*",
+                    AuthorizationResourceType.RESOURCE,
+                    Set.of(PermissionType.READ)),
+                new CreateAuthorizationRequest(
+                    ownerId,
+                    ownerType,
+                    "*",
+                    AuthorizationResourceType.PROCESS_DEFINITION,
+                    Set.of(
+                        PermissionType.READ_PROCESS_DEFINITION,
+                        PermissionType.READ_PROCESS_INSTANCE)),
+                new CreateAuthorizationRequest(
+                    ownerId,
+                    ownerType,
+                    "*",
+                    AuthorizationResourceType.DECISION_REQUIREMENTS_DEFINITION,
+                    Set.of(PermissionType.READ)),
+                new CreateAuthorizationRequest(
+                    ownerId,
+                    ownerType,
+                    "*",
+                    AuthorizationResourceType.DECISION_DEFINITION,
+                    Set.of(
+                        PermissionType.READ_DECISION_DEFINITION,
+                        PermissionType.READ_DECISION_INSTANCE))),
+            properties.getOperate() + ":write:*",
+            List.of(
+                new CreateAuthorizationRequest(
+                    ownerId,
+                    ownerType,
+                    "*",
+                    AuthorizationResourceType.MESSAGE,
+                    Set.of(PermissionType.READ)),
+                new CreateAuthorizationRequest(
+                    ownerId,
+                    ownerType,
+                    "*",
+                    AuthorizationResourceType.BATCH_OPERATION,
+                    Set.of(PermissionType.READ, PermissionType.CREATE)),
```

Comment:
see https://docs.google.com/document/d/1XqG9Y9SXvakwcJ7ZheE8lgd6X0PMWSuyH4IbYTzLYeE/edit?pli=1&tab=t.0#bookmark=id.5d04v6yihmgb
```suggestion
                    Set.of(PermissionType.READ, PermissionType.CREATE, PermissionType.UPDATE)),
```